cabot/metricsapp/models/base.py

```diff
@@ -123,3 +123,15 @@ def set_fields_from_grafana(self, fields):
             self.high_alert_value = high_alert_value
 
         self.save()
+
+    def get_status_image(self):
+        """Return a Grafana png image for the check if it exists"""
+        if self.grafana_panel is not None:
```

Comment:
I think I considered this when I wrote all this originally but then we'd be in a weird position where we'd have repeat definitions for ElasticsearchStatusCheck (for example) `ElasticsearchStatusCheck(MetricsStatusCheckBase)` and `ElasticsearchStatusCheck(GrafanaCheckBase)` that are exactly the same but inherit from different things (and we'd have to create separate forms/views for each also). I thought it'd be easier to just add an extra field to indicate whether a check was a Grafana check or not, esp since everything we're planning on adding to MetricsStatusCheckBase will have a Grafana implementation (CloudWatch does). The base StatusCheck class is still Grafana-independent also.

At this point the schema + data migration + changes to models/forms/views to change this would be pretty rough, definitely can't get it done before tomorrow :P I can open a ticket if you want though